packages/core/src/specs/v1/actionExample.ts

```
@@ -0,0 +1,91 @@
+    return { text: '' } as Content;
+  }
+
+  return {
+    text: content.text || '',
+    // V2 uses 'actions' array, V1 might use 'action' string
+    action:
+      Array.isArray(content.actions) && content.actions.length > 0 ? content.actions[0] : undefined,
+    // Copy all other properties
+    ...Object.entries(content)
+      .filter(([key]) => !['text', 'actions', 'action'].includes(key))
+      .reduce((obj, [key, value]) => ({ ...obj, [key]: value }), {}),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)